browser/ui/content_settings/brave_content_setting_image_models.cc

```diff
@@ -4,8 +4,12 @@
 
 #include "brave/browser/ui/content_settings/brave_content_setting_image_models.h"
 
-#include "brave/browser/ui/content_settings/brave_widevine_blocked_image_model.h"
 #include "brave/browser/ui/content_settings/brave_autoplay_blocked_image_model.h"
+#include "third_party/widevine/cdm/buildflags.h"
+
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
```

Comment:
New build flag introduced in C71 which was causing build problems only for Linux.